### PR TITLE
docs: update security policy; still points to edX's

### LINKED ISSUE #3: Update wording of health files for openedx org

### PR DESCRIPTION
**Pull request by kdmccormick**
Fixes #3

### CODE REVIEW

**Comment by kdmccormick** [comment]
@feanil how does this look?

File: SECURITY.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# edX Security Policy
+# Open edX Security Policy
 
-Please refer to the official security policy located [here](https://www.edx.org/policy/security)
+If you have any questions or concerns relating to the security of Open edX code (including any potential vulnerability disclosures), please follow the guidance in the [edX security policy](https://www.edx.org/policy/security). edX's security team will be happy to assist you.
+
+In the future, the Open edX project will develop an independent security team and disclosure intake process.
```
Comment (feanil) [inline SECURITY.md:5]:
```suggestion
In the future, the Open edX project will develop an independent security policy and process.
```

**Review by feanil — APPROVED** [review]
Generally looks good, one change in wording to be more general since we don't yet know how exactly the open edx security process is going to look.

File: SECURITY.md (hunk 1 above)
Comment (kdmccormick) [reply SECURITY.md:5]: Makes sense 👍🏻

**Pull request merged**